From 6a2af25d48ffddde92cff22bc67bcab4cefe06de Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 18 Jul 2017 19:23:12 +0200
Subject: appDisplay: Define constant before using it

This is now a fatal error with current gjs, however I wonder: Did this
ever work?

https://bugzilla.gnome.org/show_bug.cgi?id=785084
---
 js/ui/appDisplay.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index b3970ae..c304075 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -45,6 +45,9 @@ const INDICATORS_BASE_TIME = 0.25;
 const INDICATORS_ANIMATION_DELAY = 0.125;
 const INDICATORS_ANIMATION_MAX_TIME = 0.75;
 
+const VIEWS_SWITCH_TIME = 0.4;
+const VIEWS_SWITCH_ANIMATION_DELAY = 0.1;
+
 // Follow iconGrid animations approach and divide by 2 to animate out to
 // not annoy the user when the user wants to quit appDisplay.
 // Also, make sure we don't exceed iconGrid animation total time or
@@ -57,9 +60,6 @@ const INDICATORS_ANIMATION_MAX_TIME_OUT =
 
 const PAGE_SWITCH_TIME = 0.3;
 
-const VIEWS_SWITCH_TIME = 0.4;
-const VIEWS_SWITCH_ANIMATION_DELAY = 0.1;
-
 const SWITCHEROO_BUS_NAME = 'net.hadess.SwitcherooControl';
 const SWITCHEROO_OBJECT_PATH = '/net/hadess/SwitcherooControl';
 

